src/main/kotlin/com/plus/taxiapp/domain/MemberService.kt

```diff
@@ -0,0 +1,12 @@
+package com.plus.taxiapp.domain
+
+import org.springframework.stereotype.Service
+
+@Service
+class MemberService(
+    private val memberRepository: MemberRepository,
+) {
+    fun updateDefaultPayment(memberId: Long, type: PaymentType, paymentId: Long):Member {
```

Comment:
🚫 **[ktlint]** <standard:function-return-type-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single space expected between colon and return type